web/config/initializers/shopify_app.rb

```diff
@@ -14,7 +14,6 @@
   config.after_authenticate_job = false
   config.api_version = ShopifyAPI::AdminVersions::LATEST_SUPPORTED_ADMIN_VERSION
   config.shop_session_repository = "Shop"
-  config.user_session_repository = "User"
```

Comment:
Will `shopify_app` work without a configured user session repository?

Reply:
yes, as per [the docs](https://github.com/Shopify/shopify_app/blob/main/docs/shopify_app/session-repository.md#migrating-from-shop-based-to-user-based-token-strategy), adding a `user_session_repository` is what makes the gem use online tokens.